testing/shunit2/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Olliver Schinagl <oliver@schinagl.nl>
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=shunit2
+pkgver="2.1.7"
+pkgrel=0
+pkgdesc="shUnit2 is a xUnit based unit test framework for Bourne based shell scripts."
```

Comment:
```suggestion
pkgdesc="A xUnit based unit test framework for Bourne based shell scripts."
```

Reply:
I just copy pasted the description from the website, but I do agree that this reads nicer.